src/xrServerEntities/script_ini_file.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "script_token_list.h"
+#include "xrScriptEngine/Functor.hpp"
```

Comment:
Using this header looks bad, maybe it may affect compilation speed...

We can just use lambda script_ini_file_script.cpp :)